lib/src/common.dart

```diff
@@ -58,3 +60,65 @@ void dlog(dynamic msg) {
     print(msg.toString());
   }
 }
+
+class SpeechOptions {
+  String voice;
+  double speed;
+  String textFormat;
+  String audioFormat;
+
+  SpeechOptions(
+      {this.voice = kDefaultSpeechSynthesisVoice,
+      this.speed = kDefaultSpeechSynthesisSpeed,
+      this.textFormat = kDefaultSpeechSynthesisTextFormat,
+      this.audioFormat = kDefaultSpeechSynthesisAudioFormat});
+
+  Map<String, dynamic> toJson() {
+    return {
+      'voice': voice,
+      'speed': speed,
+      'text_format': textFormat,
+      'audio_format': audioFormat,
+    };
+  }
+}
+
+class TranscriptionOptions {
+  bool emails;
+  bool dates;
+  bool years;
+  bool domains;
+  bool urls;
+  bool amounts;
+  bool measurements;
+  bool percentages;
+  bool numbers;
+  bool ordinals;
+
+  TranscriptionOptions(
+      {this.emails = true,
+      this.dates = true,
+      this.years = true,
+      this.domains = true,
+      this.urls = true,
+      this.amounts = true,
+      this.measurements = true,
+      this.percentages = true,
+      this.numbers = true,
+      this.ordinals = true});
```

Comment:
Well, since it's Dart, it's either a rigid data class or simply a typed Map, but this would not make it clear what keys are supported. I suppose I'm fine with a typed map and simply using comments to describe the supported keys.